cluster/kubevirtci.sh

```diff
@@ -14,13 +14,15 @@
 
 export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.16'}
 
+KUBEVIRTCI_VERSION='master'
```

Comment:
wouldn't you like to set 'master' as default or use the value provided by the initiator of the script similar to KUBEVIRT_PROVIDER ?

Reply:
Not initially, but that's better, thanks.